From 9cf92f5e916a50c16d589568b9dc7ddb29555b90 Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Mon, 15 Dec 2014 16:11:09 +0800
Subject: [PATCH 479/505] MLK-9998 fix the memtool and mmdc build issue on
 Dizzy

The reason is that the macro $CC includes the CPU options
(-matc, -mfloat-abi , etc) and path of target rootfs while
$LINK in Makefile does not. Thus using $CC instead of $LINK

Signed-off-by: Zhu Jun <r01007@freescale.com>
Signed-off-by: Jason Liu <r64343@freescale.com>
---
 test/memtool/Makefile | 2 +-
 test/mmdc/Makefile    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/memtool/Makefile b/test/memtool/Makefile
index 890290c..0b1a2d2 100644
--- a/test/memtool/Makefile
+++ b/test/memtool/Makefile
@@ -21,7 +21,7 @@ CFLAGS+= -Os
 all : $(TARGET)
 
 $(TARGET):$(OBJ)
-	$(LINK) -o $(TARGET) $(OBJ) -Os
+	$(CC) -o $(TARGET) $(OBJ) -Os
 	$(STRIP) $(TARGET)
 
 .PHONY: clean
diff --git a/test/mmdc/Makefile b/test/mmdc/Makefile
index 10eaacc..50d0230 100644
--- a/test/mmdc/Makefile
+++ b/test/mmdc/Makefile
@@ -19,7 +19,7 @@ LDFLAGS+= -lstdc++
 all : $(TARGET)
 
 $(TARGET):$(OBJ)
-	$(LINK) -o $(TARGET) $(OBJ) $(LDFLAGS) -Os
+	$(CC) -o $(TARGET) $(OBJ) $(LDFLAGS) -Os
 	$(STRIP) $(TARGET)
 
 .PHONY: clean
-- 
1.8.0

